app/js/views/tableView.js

```diff
@@ -49,37 +53,74 @@ export default class TableView extends View {
       sortKey: '',
       propField: ''
     };
-    this.pageSize = 25;
+
+    this.searchDelay = 500;
+    this.searchTimeout = undefined;
 
     if (this.childview) {
       this.parentProperty = this.schema.get('parent') + '_id';
     }
 
-    this.collection.fetch().then(() => {
-      this.render();
-      if ( this.polling ) {
-        this.collection.startLongPolling();
-      }
-      this.listenTo(this.collection, 'update', this.render);
-    }, (...param) => {
-      this.errorView.render(param[1]);
-    });
+    if (this.collection !== undefined) {
+      this.collection.getPage().then(() => {
+        this.searchQuery.propField = $('select.search').val();
```

Comment:
It should be like
`this.$('select.search')`
or
`$('select.search, $el)`
Otherwise it affects multiple elements if it has multiple child resources.